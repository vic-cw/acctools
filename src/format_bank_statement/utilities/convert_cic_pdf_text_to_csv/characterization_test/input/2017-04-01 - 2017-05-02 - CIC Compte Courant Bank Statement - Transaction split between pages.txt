                                                       Crédit Industriel et Commercial

                                                                                                 CIC PARIS SAINT MARTIN

                                                                                     224 RUE SAINT MARTIN 75010 PARIS
                                                ¤      u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                            12345@cic.fr  BIC : CMCIFRPP

                                                                                                                                                                                                      41234

                                                                                                                                                                                                      12345

                                                                                                         M  JOHN  DOE

                                                                                                         3  BROADWAY     AVENUE

                                                                                                         01100      MARSEILLE

...........................................................................................................................V..O...T.R..E...C..O..N..S..E..I.L..L..E..R..:..M...S..é..b..a.s..t.i.e.n...D..U..R..A..N.D......................

                                          RELEVE ET INFORMATIONS BANCAIRES                                                                        Agence 12345

                                          02 mai 2017

                                                C/C CONTRAT PERSONNEL GLOBAL N° 00012345678 en euros (GD)

                                                TITULAIRE(S) : M JOHN SMITH

                                          Date        Date valeur   Opération                                                      Débit euros    Crédit euros

                                                                    SOLDE CREDITEUR AU 31/03/2017                                                                                                     1.234,56                                                                                                  Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                          03/04/2017  03/04/2017    VIR SEPA PROUT                                                   500,00

                                          03/04/2017  03/04/2017    PAIEMENT CB      3103 BOBIGNY                                         23,56                                                                                                  d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé

                                                                    PHARMACIE ABCDE    CARTE 00123456

                                          03/04/2017  03/04/2017    PAIEMENT CB      0104 ROSNY                                          39,51

                                                                    NATURALIA        CARTE 00123456

                                          03/04/2017  03/04/2017    PAIEMENT CB      3003 POISSY                                           46,00

KV.20170503.012345.6789.1011.12131 X 0 2                            O SELLER        CARTE 00123456

                                          (GE) : protégé par la Garantie de l'Etat                                                                <<Suite au verso>>

                                          (GD) : protégé par la Garantie des Dépôts

                                          www.garantiedesdepots.fr

                                                                                                                                                                                                      Page 1

                                            CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                          6, avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                          M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
                                          RELEVE ET INFORMATIONS BANCAIRES                                                               Agence 12345

                                          02 mai 2017

                                                C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                en  euros  (GD)

                                                TITULAIRE(S) : M JOHN SMITH

                                          Date        Date valeur   Opération                                               Débit euros  Crédit euros

                                          06/04/2017  06/04/2017    PAIEMENT CB      0504 TORCY                             6,76

                                                                    CARREFOUR        CARTE 00123456

                                          10/04/2017  10/04/2017    FRAIS PAIE CB OP 76,00        USD                       0,23

KV.20170503.012345.6789.1011.12131 X 0 V  19/04/2017  19/04/2017    PAIEMENT PSC 1504      CANNES                           10,21

                                                                    PROUT DISTR        CARTE 00123456

                                          24/04/2017  24/04/2017    PAIEMENT PSC 2104      BUSSY                            5,59

                                          (GE) : protégé par la Garantie de l'Etat

                                          (GD) : protégé par la Garantie des Dépôts

                                          www.garantiedesdepots.fr

                                                                                                                                         Page 2
                                                        Crédit Industriel et Commercial

                                                                                                    CIC PARIS SAINT MARTIN

                                                                                      224 RUE SAINT MARTIN 75010 PARIS
                                                     ¤  u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                            12345@cic.fr   BIC : CMCIFRPP

                                          RELEVE ET INFORMATIONS BANCAIRES                                                                                 Agence 12345

                                          02 mai 2017

                                                C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                            en  euros (GD)

                                                TITULAIRE(S) : M JOHN SMITH

                                          Date          Date valeur  Opération                                                         Débit euros         Crédit euros

                                                                     SAINT GEORGES       CARTE 00123456

                                          25/04/2017    25/04/2017   INCOMING FUNDS                                                                                                  1.234,56

                                                                     REASON         /A01B23456789101

                                          25/04/2017    25/04/2017   VIR ULYSSES TECHNOLOGIES (PT)                                                                                                     320,10

                                          Réf : 004                  SOLDE CREDITEUR AU 02/05/2017                                                                                                    1.234,56                                                                                                  Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                          QXBAN : AB01 CMCI FRPP XXX CDE2 FGHI 3JKL MNOP 3QR                                IBAN  :  FR76  0123 4567 8910  1112 1314 151

                                            SITUATION DE VOS CREDITS en euros au 02 mai 2017                                                                                                                                                     d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé

                                          Numéro                     Libellé du compte                                            Montant emprunté                                                    Solde (*)

                                          00012345678   ALLURE LIBRE                                                        EUR            +0,00                                                    +0,00

                                          00012345679   ETALIS                                                              EUR            +0,00                                                  +0,00

                                          (*) Solde correspondant à l'amortissement normal du prêt

                                          AUTORISATIONS DE DECOUVERTS

                                          N° de compte               Libellé                                                                               Montant euros

KV.20170503.012345.6789.1011.12131 X 0 R  00012345678   DECOUVERT C. PERS. GLOBAL TAEG 15,12% AU 26/04/2017                                                                                           123,45

                                          TOTAL MENSUEL DES FRAIS DEBITES SUR PRODUITS ET SERVICES BANCAIRES

                                          N° de compte               Libellé                                                                               Montant euros

                                          00012345678   TOTAL DES FRAIS EN MAR 2017                                                                                                                   999,70

                                          00012345678   TOTAL DES FRAIS EN AVR 2017                                                                                                                   0,01

                                          (GE) : protégé par la Garantie de l'Etat

                                          (GD) : protégé par la Garantie des Dépôts

                                          www.garantiedesdepots.fr

                                                                                                                                                                                                      Page 3

                                            CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                          6, avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                          M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
