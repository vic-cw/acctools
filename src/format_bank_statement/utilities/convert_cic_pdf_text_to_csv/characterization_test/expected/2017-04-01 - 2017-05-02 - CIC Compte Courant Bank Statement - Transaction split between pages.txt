2017-04-03,2017-04-03,"VIR SEPA PROUT","-500,00"
2017-04-03,2017-04-03,"PAIEMENT CB      3103 BOBIGNY PHARMACIE ABCDE    CARTE 00123456","-23,56"
2017-04-03,2017-04-03,"PAIEMENT CB      0104 ROSNY NATURALIA        CARTE 00123456","-39,51"
2017-04-03,2017-04-03,"PAIEMENT CB      3003 POISSY O SELLER        CARTE 00123456","-46,00"
2017-04-06,2017-04-06,"PAIEMENT CB      0504 TORCY CARREFOUR        CARTE 00123456","-6,76"
2017-04-10,2017-04-10,"FRAIS PAIE CB OP 76,00        USD","-0,23"
2017-04-19,2017-04-19,"PAIEMENT PSC 1504      CANNES PROUT DISTR        CARTE 00123456","-10,21"
2017-04-24,2017-04-24,"PAIEMENT PSC 2104      BUSSY SAINT GEORGES       CARTE 00123456","-5,59"
2017-04-25,2017-04-25,"INCOMING FUNDS REASON         /A01B23456789101","1.234,56"
2017-04-25,2017-04-25,"VIR ULYSSES TECHNOLOGIES (PT)","320,10"
